<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerDialog</class>
 <widget class="QWidget" name="ColorPickerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorPicker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="wcolor" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leColor">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string notr="true">H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QColorPickerSlider" name="hueSlider">
           <property name="maximum">
            <number>359</number>
           </property>
           <property name="value">
            <number>180</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbHue">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>359</number>
           </property>
           <property name="value">
            <number>180</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string notr="true">S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QColorPickerSlider" name="saturationSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbSaturation">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string notr="true">V</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QColorPickerSlider" name="brightnessSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbBrightness">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorPickerSlider</class>
   <extends>QSlider</extends>
   <header>src/control/qcolorpicker_slider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>hueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbHue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbSaturation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>brightnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbBrightness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
